internal/dockerutil/image.go

```diff
@@ -232,7 +240,7 @@ type Container struct {
 // A non-zero status code returns an error.
 //
 // Wait implicitly calls Stop.
-func (c *Container) Wait(ctx context.Context) (stdout, stderr []byte, err error) {
+func (c *Container) Wait(ctx context.Context, tail uint64) (stdout, stderr []byte, err error) {
```

Comment:
Explaining what the arg "tail" is for in the doc comment would be useful.

Reply:
done